docs/training/apis-howto/api-pytorch-lightning-ug.rst

```diff
@@ -105,6 +105,12 @@ In addition, we also patched some ``LightningModule`` methods to make porting yo
    Determined will automatically log the metrics you return from ``training_step`` and
    ``validation_step`` to Tensorboard.
 
+.. note::
+
+   Determined environment images no longer contain PyTorch Lightning. To use PyTorch Lightning, you
+   have to install it by adding a line like the following to ``startup-hooks.sh``: ``pip install
```

Comment:
It would be better to have the line to be added in a codeblock instead of inline.